lib/features/home_view/widgets/nav_actions_section.dart

```diff
@@ -0,0 +1,86 @@
+import "package:flutter/material.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+
+import "../../../theme/app_theme.dart";
+import "../../../utils/context_extensions.dart";
+import "../../navigator/utils/navigation_commands.dart";
+
+class NavActionsSection extends ConsumerWidget {
+  const NavActionsSection({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) => Center(
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+          children: [
+            _NavActionButton(
+              context.localize.sks_menu,
+              Icon(
+                Icons.restaurant_menu,
+                color: context.colorTheme.whiteSoap,
+                size: 32,
+              ),
+              ref.navigateToSksMenu,
+            ),
+            _NavActionButton(
+              context.localize.parkings_title,
+              Icon(
+                Icons.directions_car,
+                color: context.colorTheme.whiteSoap,
+                size: 32,
+              ),
+              ref.navigateParkings,
+            ),
+            _NavActionButton(
+              context.localize.about_us,
+              Image.asset(
+                "assets/png/splash_screen/splash_logo.png",
```

Comment:
so we're actually using flutter gen package that generates our paths from the assets, so instead of this string, you can just use
```dart
Assets.png.splashScreen.splashLogo.path,
```

it should be generated and available, just import the Assets object